settings.gradle

```diff
@@ -7,4 +7,5 @@ include ':commonui',
         ':transferrepository',
         ':transferui',
         ':userrepository',
+        ':userbalancerepository',
```

Comment:
How will you get prepaid card balance?
Look at iOS PR, he is creating  `BalanceRepository` which then has `PrepaidCardBalanceRepository` and `UserBalanceRepository`

Reply:
prepaid card balance in transfermethodrepository 
<img width="594" alt="Screenshot 2021-03-10 at 12 42 20 PM" src="https://user-images.githubusercontent.com/71114945/110590755-36d9ee80-819e-11eb-862d-c4109734a1a4.png">

refactor prepaid card balance transfermethodrepository  to balance repository ?

Reply:
added PrepaidCardBalanceRepository in UI - SDK